src/ophyd_async/epics/_backend/_aioca.py

```diff
@@ -15,7 +16,7 @@
     caput,
 )
 from aioca.types import AugmentedValue, Dbr, Format
-from bluesky.protocols import Descriptor, Dtype, Reading
+from bluesky.protocols import DataKey, Dtype, Reading
```

Comment:
Probably want to make this change in other places, and change the `SignalBackend.descriptor` method to `.datakeys` as well. That should be another ticket though, captured in #145